lib/util/const.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+const progress = Center(child: CircularProgressIndicator());
+
+class StorageKeys{
+  static String humhubInstance = "humHubInstance";

```

Comment:
@PrimozRatej Out of interest: Does the variable `humHubInstance` contain the manifest?  Since such storage variables are probably difficult to change later, I would perhaps use something like: `manifest` and `lastUrl` without HumHub. 

If you want to use HumHub im vars, please completely lowercase `humhub` and not camelcase `humHub`.

Thanks :-)